app/workers/receive_base.rb

```diff
@@ -0,0 +1,33 @@
+module Workers
+  class ReceiveBase < Base
+    sidekiq_options queue: :receive
+
+    include Diaspora::Logging
+
+    # don't retry for errors that will fail again
+    def filter_errors_for_retry
```

Comment:
yes, I know, but after removing the deprecated errors, it will be under 20 lines again. I hope that is OK for you, holy @diaspora-code-review :innocent: